support/_index.html

```diff
@@ -60,9 +60,7 @@ <h2>Community Support</h2>
 <div class="atlnavitem">
 <h2>Professional Support</h2>
 <div class="contents">
-<p><p>The ATL team can be contacted through the official Eclipse forums and newsgroup. Yet this form of support may not meet daily needs of an enterprise adopter as far as response delays are involved. 
-<br /><br />Want to use ATL in a commercial environment and need training, reviews or simply an expert to get it done? We would be glad to assist you.
-atl is the result of several man-years of R&D started in the French company <a href="https://www.obeosoft.com/">Obeo</a> whom developers could offer you the best support.
+<p><p>The ATL team can be contacted through the official Eclipse forums and newsgroup. Yet this form of support may not meet daily needs of an enterprise adopter as far as response delays are involved.
```

Comment:
Also remove "Yet this form of support may not meet daily needs of an enterprise adopter as far as response delays are involved."